src/compositedataframe.jl

```diff
@@ -43,14 +43,18 @@ A constructor of an `AbstractCompositeDataFrame` that mimics the `DataFrame`
 constructor.  This returns a composite type (not immutable) that is an
 `AbstractCompositeDataFrame`.
 
-This uses `eval` to create a new type within the current module.
+This uses `eval` to create a new type within the module specified by the
+inmodule keyword.
```

Comment:
"keyword *argument"*. Use backticks around `inmodule`. Should also mention `inmodule = DataFramesMeta` before `kwargs...` in signatures.